features/steps/common.py

```diff
@@ -27,6 +28,7 @@ def i_click_the_first_participant_journal_button(context):
 
 @when(u'I click the link "{text}"')
 def i_click_the_link(context, text):
+    time.sleep(1)
```

Comment:
@thraxil has a good pattern for timing related failures. The approach is to try the test first, catch the exception, sleep and retry. This circumvents slowdowns when the initial attempt is likely to succeed. 
Example: https://github.com/ccnmtl/mediathread/blob/master/terrain.py#L304